nncf/onnx/graph/model_transformer.py

```diff
@@ -58,9 +58,12 @@ def _get_target_edge(self, port_id: int, node_name: str, transform_type: TargetT
         """
         if transform_type in [TargetType.PRE_LAYER_OPERATION, TargetType.OPERATION_WITH_WEIGHTS]:
             return onnx_graph.get_node_edge_names(node_name)['input'][port_id]
+        input_edges = set()
         if node_name in nncf_input_node_next_nodes:  # ADD INPUT NODE CASE
-            node_names = nncf_input_node_next_nodes[node_name]
-            input_edges = set(onnx_graph.get_node_edge_names(name)['input'][port_id] for name in node_names)
+            next_nodes = nncf_input_node_next_nodes[node_name]
+            for next_node in next_nodes:
+                name, port_id = next_node[0], next_node[1]
```

Comment:
IMHO, it's better to use `dict` inside, instead of mapping by indexes of very different types of data.

Reply:
Discussed